config/locales/en/brexit_checker/results.yml

```diff
@@ -8,7 +8,8 @@ en:
       description_no_actions: |
         Based on your responses, you do not need to take any action to prepare for the new rules from 1 January 2021.
       description_no_answers: |
-        To change your answers, follow the link below. This is a safe and secure service. We do not store any of your personal information.
+        To change your answers, follow the link below.
```

Comment:
This won't keep the line breaks:

<img width="648" alt="Screenshot 2020-01-30 at 15 41 23" src="https://user-images.githubusercontent.com/29889908/73464516-fc850780-4376-11ea-8cff-5d70334fd873.png">